docs/resources/covid-19.md

```diff
@@ -104,99 +212,505 @@ From the response of Safe Place API, you will get an overall safety score and sc
+    },
+    "dataSources": {
+      "healthDepartmentSiteLink": "http://www.kdca.go.kr/index.es?sid=a3",
+      "governmentSiteLink": "http://ncov.mohw.go.kr/en/"
+    },
 ```
+
+If available, the disease infection information is suplemented by a link to the infection statistics map, for example:
+
+```json
+"infectionMapLink": "https://www.nytimes.com/interactive/2021/us/covid-cases.html"
+```
+
 And what is the vaccination ratio in this country?
 
 ```json
-        "areaVaccinated": [
-            {
-                "date": "2022-02-27",
-                "vaccinationDoseStatus": "oneDose",
-                "percentage": 87.336
-            },
-            {
-                "date": "2022-03-09",
-                "vaccinationDoseStatus": "fully",
-                "percentage": 86.56
-            }
+    "areaVaccinated": [
+      {
+        "lastUpdate": "2022-05-16",
+        "vaccinationDoseStatus": "oneDose",
+        "percentage": 87.698
+      },
+      {
+        "lastUpdate": "2022-05-30",
+        "vaccinationDoseStatus": "fully",
+        "percentage": 86.9
+      }
+    ]
 ```
 
 
 ### Get Travel Restrictions information 
 
-From the response of Travel Restrictions API: 
+From the response of Travel Restrictions API, you can get information on:
 
-Entry information, which origin countries will be required to quarantine, entry requirements, mask information, tracing application, and much more...
+#### Area restrictions
+
+These are the general restrictions in the area:
 
 ```json
-            "entry": {
-                "date": "2022-03-04",
-                "text": "<p>All inbound non-resident foreign nationals must secure a visa before their travel. All travelers, including residents and Non- South Korean nationals, must hold a valid negative PCR test result to be allowed to board an inbound flight to South Korea. However, South Korean nationals who were found after entry with a non-conforming certificate must undergo a five days quarantine in a government-designated facility and an additional two days home quarantine.</p>",
-                "ban": "No",
-                "throughDate": "indef",
-                "rules": "http://ncov.mohw.go.kr/duBoardList.do?brdId=2&brdGubun=23",
-                "borderBan": [
-                    {
-                        "borderType": "maritimeBorderBan",
-                        "status": "Closed"
-                    },
-                    {
-                        "borderType": "landBorderBan",
-                        "status": "Closed"
-                    }
-                ]
-            },
+    "areaRestrictions": [
+      {
+        "lastUpdate": "2022-05-26",
+        "text": "<p>Capacity limits on public gatherings and time restrictions for bars, restaurants and other venues have been lifted. Houses of worship may also operate at full capacity.</p>",
+        "restrictionType": "OTHER"
+      }
+    ]
 ```
 
+#### Area access restrictions
```

Comment:
What's the difference with "Area restrictions"? Can we write a small sentence to explain it?